types/jsonwebtoken/index.d.ts

```diff
@@ -166,4 +166,4 @@ declare function verify(
 declare function decode(
     token: string,
     options?: DecodeOptions,
-): null | object | string;
+): null | { [key: string]: any } | string;
```

Comment:
Why change from `object`?

Reply:
Using `object` is equivalent to `{}`, using `{ [key: string]: any }` describe the general shape of the object and so makes it less restrictive.

Reply:
> Using `object` is equivalent to `{}`, using `{ [key: string]: any }` describe the general shape of the object and so makes it less restrictive.
  
you do not need the `| string` then.. it does match `{ [key: string]: any } `

Reply:
By less restrictive, you mean it's not type-checked? Can't you leave the declaration as `object` nd cast it to the type you expect?